whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -111,4 +114,13 @@ public void stop(final boolean force) {
             LOGGER.error("Stopping server", e);
         }
     }
+
+    // Log requests to org.eclipse.jetty.server.RequestLog
+    private RequestLog createRequestLog() {
+        final Slf4jRequestLog requestLog = new Slf4jRequestLog();
+        requestLog.setExtended(true);
+        requestLog.setLogTimeZone("GMT");
```

Comment:
Are our current http log files in UTC or CEST?

Reply:
Our current log files are in CET or CEST (i.e. local time). I updated the code to specify local time (the default is GMT).